template/frontend/src/App.test.js

```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react'
-import App from './App'
-
-test('renders learn react link', () => {
-  render(<App />)
-  const linkElement = screen.getByText(/learn react/i)
-  expect(linkElement).toBeInTheDocument()
-})
```

Comment:
let's leave this in as well as the `react-scripts test` (in package.json) because some users may want to do static testing still